contracts/FlowToken.cdc

```diff
@@ -83,6 +84,64 @@ pub contract FlowToken: FungibleToken {
                 FlowToken.totalSupply = FlowToken.totalSupply - self.balance
             }
         }
+
+        /// Get all the Metadata Views implemented by FlowToken
+        ///
+        /// @return An array of Types defining the implemented views. This value will be used by
+        ///         developers to know which parameter to pass to the resolveView() method.
+        ///
+        pub fun getViews(): [Type]{
```

Comment:
Between `getViews()` and `resolveView()`, don't we want `Vault` to implement `MetadataViews.Resolver`?

Reply:
We can't add interface conformances in an upgrade, so we can't do that here unfortunately

Reply:
[Docs](https://developers.flow.com/cadence/language/contract-updatability#structs-resources-and-interfaces) mention adding an interface conformance is a valid upgrade. Is the issue here adding a new conformance?